prototype/Character/Player/Player.gd

```diff
@@ -215,10 +217,20 @@ func _unhandled_input(event):
 		else:
 			get_tree().change_scene('res://prototype/Main.tscn')
 
+	
+func _on_body_entered(body):
+	._on_body_entered(body)
+	
+	if self.is_using_jetpack and body.is_in_group("floor"):
```

Comment:
`body.is_in_group("floor")`: While this is good enough for this prototype, imagine hitting a ceiling in a multi-level building. This will have to be re-done properly if this prototype lives longer than expected.

Reply:
After spending several hours on this, this is the only solution I could figure out that worked.